waspc/src/Wasp/Operation.hs

```diff
@@ -0,0 +1,25 @@
+module Wasp.Operation
+    ( Operation(..)
+    , getName
+    , getJsFn
+    ) where
+
+-- TODO: Is this ok approach, should I instead use typeclass?
```

Comment:
Still not sure about this hm!

Reply:
hm hm yes, this will make a nice blog post I can see it already